parser.js

```diff
@@ -85,11 +82,11 @@ function setData(){
 
 	
 	var tempString="";
-	tempString=capFirst(tempData[0]);
+	tempString=tempData[0];
 	j=1;
 	if (tempData.length > 1){

```

Comment:
Provide a better variable name for this while look instead of "j", do the same for the variable "i" just above it.